config/rbac/submariner-route-agent/cluster_role.yaml

```diff
@@ -51,3 +51,11 @@ rules:
       - delete
       - update
       - deletecollection
+  - apiGroups:
+      - operator.tigera.io
+    resources:
+      - installations
+    verbs:
+      - get
+      - patch
```

Comment:
@tpantelis QQ: checking your comment on submariner PR [1], do we need the patch verb here? 

[1] 
https://github.com/submariner-io/submariner/pull/3570#discussion_r2270101324